### PR TITLE
Add getErrorSummary().

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Tests pass?   | ✔️

[x] `Add getErrorSummary()` need for Html Forms.

### CODE REVIEW

File: tests/Stubs/LoginForm.php [hunk 1]
```diff
@@ -86,4 +87,14 @@ private function loginRules(): array
             new Email()
         ];
     }
+
+    private function passwordRules(): array
+    {
+        return [
+            new Required(),
+            (new HasLength())
```
Comment (samdark) [inline tests/Stubs/LoginForm.php:95]: This rule should be removed. It's a bad practice to limit password length.

Comment (terabytesoftw) [reply tests/Stubs/LoginForm.php:95]:
```
        return [
            new Required(),
            (new HasLength())
            ->min(8)
            ->tooShortMessage('Is too short.'),
        ];
```

Comment (terabytesoftw) [reply tests/Stubs/LoginForm.php:95]: done.

**Comment by samdark** [comment]
👍

**Pull request merged**